src/kmeans.py

```diff
@@ -0,0 +1,154 @@
+import matplotlib.pyplot as plt
+import os
+import networkx as nx
+import numpy as np
+from sklearn.cluster import KMeans
+from matplotlib import cm
+
+
+os.chdir(f"c:/Users/dmitr/Documents/git/node2vec")
+
+SWITCH = "str_eq" # homophily, str_eq, struc2vec
+HOMOPHILY = False
+EDGELIST_FNAME = "graph/les_miserables_int.edgelist"
+D = 16
+P = 1
```

Comment:
Adding Q is well would be a good idea and then update it in the if statements later.